xbmc/guilib/GUITextLayout.cpp

```diff
@@ -540,75 +542,93 @@ void CGUITextLayout::WrapText(const vecText &text, float maxWidth)
   if (!m_font)
     return;
 
-  int nMaxLines = (m_maxHeight > 0 && m_font->GetLineHeight() > 0)?(int)ceilf(m_maxHeight / m_font->GetLineHeight()):-1;
-
   m_lines.clear();
 
+  size_t nMaxLines;
+  if (m_maxHeight > 0 && m_font->GetLineHeight() > 0)
```

Comment:
float comparisons with 0.0f

Reply:
this is usually optimized by compilers or you have found some CI compiler warnings?

Reply:
I don't see the relevance? m_maxHeight and GetLineHeight are floats to be compared with float literals. I know it doesn't make much difference in the end.